fast_llm/layers/language_model/config.py

```diff
@@ -118,6 +118,16 @@ class LanguageModelBaseConfig(LanguageModelArchitectureConfig, BaseModelConfig):
         hint=FieldHint.feature,
         valid=check_field(Assert.geq, 0),
     )
+    init_method_max_embed: float = Field(
```

Comment:
`float|None`

Reply:
do you also want this change in `fast_llm/layers/transformer/config.py`?

Reply:
Yes, all of them